src/api/nexusaiAPI.js

```diff
@@ -187,6 +187,19 @@ export default {
     );
   },
 
+  router: {
+    read({ projectUuid }) {
+      /* return {
+        data: {
+          content_base_uuid: '150cf2c6-e8ff-4135-a95d-e0e32aece0ba',
+          intelligence_uuid: '1a40e439-c2c1-4386-ba99-b51df1b53eae',
+        },
+      }; */
```

Comment:
This is dead code?

Reply:
It really makes sense to remove it, it's all right with this integration.

Reply:
```suggestion

```